qds_sdk/cluster.py

```diff
@@ -170,6 +170,9 @@ def _parse_create_update(cls, args, action, api_version):
         ec2_group.add_argument("--vpc-id",
                                dest="vpc_id",
                                help="vpc to create the cluster in",)
+        ec2_group.add_argument("--bastion-node-public-dns",
+                               dest="bastion_node_public_dns",
+                               help="public dns name of the bastian node. Required only if cluster is in private subnet",)
```

Comment:
typo in `help` - `bastian` => `bastion`. 
Also to be specific: `Required only if the cluster is in the private subnet of a EC2-VPC`